commit 6ee3a34140b2fc1881eeccb9a5037957b69e14c0 (refs/changes/63/4663/4)
Author: Mike Gerdts <mike.gerdts@joyent.com>
Date:   2018-08-29T22:40:17+00:00 (1 year, 1 month ago)
    
    OS-7119 adjust pci slots to support windows
    Reviewed by: Jorge Schrauwen <jorge@blackdot.be>
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Approved by: Patrick Mooney <patrick.mooney@joyent.com>

diff --git a/usr/src/lib/brand/bhyve/zone/boot.c b/usr/src/lib/brand/bhyve/zone/boot.c
index ea0f8c15a6..1adea7b26f 100644
--- a/usr/src/lib/brand/bhyve/zone/boot.c
+++ b/usr/src/lib/brand/bhyve/zone/boot.c
@@ -43,12 +43,12 @@
 
 typedef enum {
 	PCI_SLOT_HOSTBRIDGE = 0,	/* Not used here, but reserved */
-	PCI_SLOT_LPC,
-	PCI_SLOT_CD,
+	PCI_SLOT_CD = 3,		/* Windows ahci allows slots 3 - 6 */
 	PCI_SLOT_BOOT_DISK,
 	PCI_SLOT_OTHER_DISKS,
 	PCI_SLOT_NICS,
 	PCI_SLOT_FBUF = 30,
+	PCI_SLOT_LPC = 31,		/* Windows requires lpc in slot 31 */
 } pci_slot_t;
 
 static boolean_t debug;
